x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -357,7 +358,12 @@ public void testInlineStatsWithoutGroups() {
     }
 
     public void testStringAsIndexPattern() {
-        for (String command : List.of("FROM", "METRICS")) {
+        List<String> commands = new ArrayList<>();
+        commands.add("FROM");
+        if (Build.current().isSnapshot()) {
```

Comment:
Are the tests run in non-snapshot code? Can stay as is, was just wondering.

Reply:
It doesn't have the `assumeTrue` for snapshot, just be to cautious.